MatrixSDK/Data/RoomSummaryStore/CoreData/MXCoreDataRoomSummaryStore.xcdatamodeld/.xccurrentversion

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>_XCCurrentVersionName</key>
+	<string>MXRoomSummaryCoreDataStore2.xcdatamodel</string>
```

Comment:
I do not know why we haven't been versioning the `MXRoomSummaryCoreDataStore` despite plenty of updates to the model. When tested without a new version the app runs fine without crashes, even if an existing property is renamed, suggesting Core Data is doing a lot under the hood to guess mapping old data without a known model to the current one, either way this is an undocumented behaviour. For that reason it is still better to declare a new version each time the model is changed.

Reply:
Yeah you right. Thanks for adding the first new version.